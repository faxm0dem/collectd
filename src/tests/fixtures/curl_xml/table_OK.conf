LoadPlugin "curl_xml"

<Plugin "curl_xml">
  <URL "file://table.xml">
    Host "gemaster"
    Instance "GridEngine"
    <xpath "/GridEngineBasicCount/Jobs">
      Type "gauge"
      InstanceFrom "@Instance"
      ValuesFrom "Count/text()"
    </xpath>
    
    <xpath "/GridEngineBasicCount/Slots">
      Type "gauge"
      InstanceFrom "@Instance"
      ValuesFrom "Count/text()"
    </xpath>
    
    <xpath "/GridEngineBasicCount/Scheduler">
      Type "duration"
      InstanceFrom "@Instance"
      ValuesFrom "SchedDuration/text()"
    </xpath>
    <xpath "/GridEngineBasicCount/Scheduler">
      Type "gauge"
      InstanceFrom "@Instance"
      ValuesFrom "SpawnedJobsCount/text()"
    </xpath>
  </URL>
</Plugin>

