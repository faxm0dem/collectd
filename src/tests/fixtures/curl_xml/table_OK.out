gemaster/curl_xml-GridEngine/duration-ParallelJobs   seconds=2.000000e+00
gemaster/curl_xml-GridEngine/duration-SequentialJobs seconds=1.000000e+00
gemaster/curl_xml-GridEngine/gauge-ParallelJobs      value=3.000000e+00
gemaster/curl_xml-GridEngine/gauge-Pending           value=1.275000e+03
gemaster/curl_xml-GridEngine/gauge-Required          value=1.491000e+03
gemaster/curl_xml-GridEngine/gauge-Running           value=1.369400e+04
gemaster/curl_xml-GridEngine/gauge-SequentialJobs    value=2.000000e+00
